src/client.js

```diff
@@ -176,13 +176,14 @@ class Credentials {
      * @param {string} sessionToken the session token. It's exact form depends on the credentials type. For instance it can be "user/passord" for the "UserPassword" credentials type
      * @param {string} securityToken the security token. Will use an empty token if not specified
      */
-    constructor(type, sessionToken, securityToken) {
+    constructor(type, sessionToken, securityToken, bearerToken) {
```

Comment:
I’m not too fan of the changes of the signature of the Credentials object (look at line 321). If type is bearer, can we just consider that the session token is the bearer token and not add an extra parameter ? It would be good to remain consistent with the service token support and either do the same or change the service token case